df.buildings.xml

```diff
@@ -745,14 +743,11 @@
         <stl-vector name='permitted_workers' type-name='int32_t' ref-target='unit'/>
         <int32_t name='min_level'/>
         <int32_t name='max_level' init-value='3000'/>
-        <static-array type-name='int32_t' count='12'/>
+        <compound name='links' type-name='stockpile_links'/>
         <int32_t name='max_general_orders'/>
         <bool name='block_general_orders'/>
         <padding size='3'/>
         <static-array name='blocked_labors' type-name='bool' count='94' index-enum='unit_labor'/>
-    </struct-type>
-
-    <struct-type type-name='manager_profile'>
         <int32_t name='unk_v43_1' comment='v0.43.01'/>
         <static-array type-name='int8_t' name='unk_v43_2' count='98' comment='v0.43.01'/>
```

Comment:
Are you sure these two fields aren't just the same as the last four in workshop_profile?